### PR TITLE
Add GSoC 2022 instructions

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

File: _activities/gsoc.md [hunk 1]
```diff
@@ -24,24 +24,22 @@ To answer these questions, particle physicists build software to simulate and an
...
 
-The selection process in our organization was split in two phases. All candidates had to pass pre-selection evaluation tests prepared by mentors, demonstrating the skills needed for the respective projects. The successful candidates had a detailed exchange with mentors on a given project idea, and their proposals were evaluated for the final student selection. The detailed [timeline]({{site.baseurl}}/activities/gsoc.html#timeline) of these phases is shown below.
-
-The blog posts made by our 2021 students are available [here]({{ site.baseurl }}/gsoc/2021/blogs.html).
+The selection process in our organization will be split in two phases. All candidates will have to pass pre-selection evaluation tests prepared by mentors, demonstrating the skills needed for the respective projects. The successful candidates had a detailed exchange with mentors on a given project idea, and their proposals are then evaluated for the final student selection. The detailed [timeline]({{site.baseurl}}/activities/gsoc.html#timeline) of these phases is shown below.
```
Comment (agheata) [inline _activities/gsoc.md:34]:
```suggestion
The selection process in our organization will be split into two phases. All candidates will have to pass pre-selection evaluation tests prepared by mentors, demonstrating the skills needed for the respective projects. The successful candidates will have a detailed exchange with mentors on a given project idea, and their proposals are then evaluated for the final student selection. The detailed [timeline]({{site.baseurl}}/activities/gsoc.html#timeline) of these phases is shown below.
```

File: _activities/gsoc.md [hunk 2]
```diff
@@ -51,101 +49,101 @@ For further information, feel free to contact the HSF GSoC admins or join our op
...
   </tr>
   <tr>
-    <td> Apr 21 - Apr 27 </td>
+    <td> Apr 22 - Apr 29 </td>
     <td> Admins propose a range of slots to be asked</td>
   </tr>
   <tr>
-    <td> Apr 28 - May 2 </td>
+    <td> Apr 29 - May 2 </td>
     <td> HSF committee validates the range of slots and the projects in reserve </td>
```
[the remark this answers is not in the document]
Comment (agheata) [reply _activities/gsoc.md:115]: Just squash this into the previous

**Comment by hegner** [comment]
@graeme-a-stewart - can you check? to me the link errors seem purely false positives.

File: gsoc/guideline.md [hunk 3]
```diff
@@ -80,6 +80,6 @@ create a file for your organization and edit its contents as appropriate.
 
 Organization logos should be called `ORGANIZATION-logo.png` and be less than 100kB in size.
 
-## HSF GSoC Mentor Guideline 2021
+## HSF GSoC Mentor Guideline 2022
```
Comment (agheata) [inline gsoc/guideline.md:83]: We should probably add below a description of the new properties that we want proposals to contain (duration, difficulty, ...)

**Review by agheata — CHANGES_REQUESTED** [review]
Thanks! I've made, mainly suggestions for the timeline, given the lessons from last year. The guides need also some additions.

File: _activities/gsoc.md [hunk 4]
```diff
@@ -51,101 +49,101 @@ For further information, feel free to contact the HSF GSoC admins or join our op
 
 ### For Projects and Mentors
 
-Instructions for participating projects and mentors can be found [here]({{ site.baseurl }}/gsoc/guideline.html). The main differences compared to last year are that proposals have to be tuned for 175-hour project length, and the target objectives must be well-defined deliverables.
+Instructions for participating projects and mentors can be found [here]({{ site.baseurl }}/gsoc/guideline.html). The main differences compared to last year are that proposals have to be tuned for either 175-hour or 350h project length, and the target objectives must be well-defined deliverables.
```
[the remark this answers is not in the document]
Comment (hegner) [reply _activities/gsoc.md:52]: thanks for spotting. seems I missed one part to add the 350h

File: gsoc/guideline.md (hunk 3 above)
Comment (hegner) [reply gsoc/guideline.md:83]: into the text or just into the google doc?

Comment (agheata) [reply gsoc/guideline.md:83]: Just in the doc containing the instructions

**Comment by graeme-a-stewart** [comment]
> @graeme-a-stewart - can you check? to me the link errors seem purely false positives.

Yes, it seems the link checker has some trouble with the `#timeline` id tag - I checked that it works fine in the browser itself. Agreed that it's a false positive.

File: _activities/gsoc.md [hunk 5]
```diff
@@ -24,24 +24,22 @@ To answer these questions, particle physicists build software to simulate and an
...
-
-The blog posts made by our 2021 students are available [here]({{ site.baseurl }}/gsoc/2021/blogs.html).
+The selection process in our organization will be split in two phases. All candidates will have to pass pre-selection evaluation tests prepared by mentors, demonstrating the skills needed for the respective projects. The successful candidates had a detailed exchange with mentors on a given project idea, and their proposals are then evaluated for the final student selection. The detailed [timeline]({{site.baseurl}}/activities/gsoc.html#timeline) of these phases is shown below.
 
 ### For Students
 
 If you are interested in the HSF projects, please start by having a look at our [guidelines for students]({{ site.baseurl }}/gsoc/students-guideline.html).
```
[the remark this answers is not in the document]
Comment (agheata) [reply _activities/gsoc.md:38]: We may add there that this year student blogs are mandatory, many Orgs are doing this and Google strongly encourages. Last year it was not mandatory and we didn't get any on our pages.

**Review by agheata — APPROVED** [review]
The Mar 20- Apr 3 item in the timeline is now redundant and can go away, but we can do this kind of adjustments later as well.

**Pull request merged**